shared_types/src/standings.rs

```diff
@@ -101,36 +101,24 @@ impl Standings {
     }
 
     pub fn update_standings(&mut self, tiebreaker: Tiebreaker) {
-        for (i, players) in self.players_standings.clone().iter().enumerate() {
-            if players.len() > 1 {
-                self.players_standings.remove(i);
-                let mut scores: Vec<(Uuid, f32)> = players
-                    .iter()
-                    .map(|player| {
+    let mut scores: Vec<(Uuid, f32, f32)> = self.players.clone().into_iter().map(|player| {
                         (
-                            *player,
+                            player,
+                            self.get_raw_points(player),
```

Comment:
Doesn't this calculate the raw_points on the spot? They should already be calculated here.
Couldn't we just do this?
```rs
*self.players_scores.get(&player).unwrap().get(&Tiebreaker::RawPoints).unwrap_or(&0.0)
```

Reply:
This works to fix the issue, but fails the unit tests, I am fine to change this